models/gif_psc_exp.cpp

```diff
@@ -321,8 +321,6 @@ void
 nest::gif_psc_exp::update( Time const& origin, const long from, const long to )
 {
 
```

Comment:
I guess you get the point. This might actually be much easier in a local `git difftool --dir-diff` view...

Reply:
Oh well, I would have thought that clang-format actually detects and removes empty lines, but that does not seem to be the case.

Reply:
Done

Reply:
Cheers!